common/djangoapps/track/backends/tests/test_oli.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests oli.py
```

Comment:
What is `oli.py`? This docstring just looks like a repetition of the file name..